packages/platform-android/native_modules.gradle

```diff
@@ -54,7 +54,8 @@ class ReactNativeModules {
   private ArrayList<HashMap<String, String>> reactNativeModules
 
   private static String LOG_PREFIX = ":ReactNative:"
-  private static String REACT_NATIVE_CONFIG_CMD = "react-native config"
+  private static String REACT_NATIVE_CLI_BIN = "node_modules${File.separator}@react-native-community${File.separator}cli${File.separator}build${File.separator}index.js"
```

Comment:
RN can technically be in monorepo so we can't make assumptions about where node_modules with cli are. That's why in iOS implementation we spawn a Node process and use its `require.resolve` to output the resolved path to stdout

Reply:
This is handled by the final arg here: https://github.com/react-native-community/cli/blob/7a7115f6e8dd10f9eb33eb9d1e8b4c085e08a67f/packages/platform-android/native_modules.gradle#L178 - the process is started in the project root, which can also be overridden here: https://github.com/react-native-community/cli/blob/7a7115f6e8dd10f9eb33eb9d1e8b4c085e08a67f/packages/platform-android/native_modules.gradle#L114

Reply:
Ok, so it won't be seamless but at least that's supported. We need to add documentation on how the autolinking is working on how to configure it.

Reply:
@Salakar, we already have `config.root` in our config outputted from `react-native config`. You could use that one.

Also, you can just do `console.log(require.resolve('react-native'))` script and exec `node` to get the exact location. I think this is what we use on iOS.